thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -168,7 +224,8 @@ static ServiceInfo newServiceInfo(
             });
         }
 
-        return new ServiceInfo(name, functions, classes, endpoints, docStrings.get(name), exampleHeaders);
+        return new ServiceInfo(name, functions, classes, endpoints,
```

Comment:
Agreed that it's annoying to pass docstrings and examples around. How about making DocService reconstruct the specification later with the docstrings and examples so a plugin doesn't need to do that? A little bit more complexity in DocService but docstrings and example would be where they are supposed be.